src/modules/dashboard/components/ExpenditurePage/ExpenditureSettings/Streaming/LockedStreaming.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useCallback, useState } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import classNames from 'classnames';
+
+import { FormSection } from '~core/Fields';
+import { CollapseExpandButtons } from '~dashboard/ExpenditurePage/Payments';
+import { Colony } from '~data/index';
+
+import LockedFundingSource from './LockedFundingSource';
+import { FundingSource } from './types';
+import styles from './Streaming.css';
+
+const MSG = defineMessages({
+  fundingSource: {
+    id: 'dashboard.ExpenditurePage.Streaming.LockedStreaming.fundingSource',
+    defaultMessage: 'Funding source',
+  },
+  title: {
+    id: 'dashboard.ExpenditurePage.Streaming.LockedStreaming.title',
+    defaultMessage: '{nr}: {team}, {rateAmount} {rateToken} / {rateTime}',
+  },
+});
+
+const displayName = 'dashboard.ExpenditurePage.Streaming.LockedStreaming';
+
+interface Props {
+  fundingSources?: FundingSource[];
+  colony: Colony;
+}
+
+const LockedStreaming = ({ fundingSources, colony }: Props) => {
+  const [expandedFundingSources, setExpandedFundingSources] = useState<
```

Comment:
What do you think about making it extra clear this stores the indexes of funding sources rather than funding sources themselves?